iiblib/iib_authentication.py

```diff
@@ -0,0 +1,106 @@
+import os
+import kerberos
+import subprocess
+import tempfile
+
+
+# pylint: disable=bad-option-value,useless-object-inheritance
+class IIBAuth(object):
```

Comment:
@lipoja do you want to create a task for an improvement?

Reply:
Only if you and @midnightercz agrees with this change. My opinion is to change it because it looks like the IIBAuth is abstract class providing template and defining methods that should be defined in child classes.

Reply:
I'm fine with proposed changes. But in separated PR

Reply:
I will create a task for that